docker/serverAndUI/Dockerfile

```diff
@@ -51,4 +51,4 @@ RUN git clone https://github.com/Netflix/conductor.git \
 EXPOSE 5000 8080
```

Comment:
I didn't touch `docker/serverAndUI`.  I think you could probably deprecate it in favor of asking noobs to just use docker-compose to run the stack locally.